### PR TITLE
Add version number to install header

### LINKED ISSUE #194: version number/release day/time should be prominently displayed near any installer/artifacts
when people go to download things it can be helpful to let them know what version number thing they are downloading and when it was last released. this will become more important once we tackle the idea of "stability channels" some time in the future.

relevant element convo with idea from @SaraVieira :

Screen Shot 2023-02-27 at 12 38 11 PM

### PR DESCRIPTION
**Pull request by SaraVieira**
- Adds the version number to the main install header
- Adds published at date
image

closes: https://github.com/axodotdev/oranda/issues/194

### CODE REVIEW

**Review by ashleygwilliams — COMMENTED** [review]
we're fetching the github release response right? there's a `"created_at"` timestamp- could we use that?

**Comment by SaraVieira** [comment]
you are super right, I made this in the morning

Updated the description

File: oranda-css/css/theme.css [hunk 1]
```diff
@@ -172,6 +172,11 @@ div.table code {
   @apply flex gap-2 items-center flex-col mb-16;
 }
 
+
```
Comment (ashleygwilliams) [inline oranda-css/css/theme.css]: extra white space we should remove

File: src/site/artifacts/cargo_dist.rs [hunk 2]
```diff
@@ -21,15 +22,15 @@ pub fn get_os(name: &str) -> Option<&str> {
     }
 }
 
-pub fn fetch_manifest(config: &Config) -> Result<cargo_dist::DistManifest> {
+pub fn fetch_manifest(config: &Config) -> Result<(cargo_dist::DistManifest, String)> {
```
Comment (ashleygwilliams) [inline src/site/artifacts/cargo_dist.rs:26]: really not a fan of tuples, would probably make a "release metadata struct" - overkill for now but feel 100% confident we'll put stuff there in the future and i think the improved readability is worth doing it now

File: src/site/artifacts/cargo_dist.rs [hunk 3]
```diff
@@ -183,10 +184,19 @@ pub fn build(config: &Config) -> Result<Box<div<String>>> {
                     None => String::from("We couldn't detect the system you are using."),
                 };
                 let install_code_block = build_install_block(config, &typed, release, artifact);
+                let title = format!("Install v{}", release.app_version);
+                let formatted_date = match DateTime::parse_from_rfc3339(published_at.as_str()) {
```
Comment (ashleygwilliams) [inline src/site/artifacts/cargo_dist.rs:189]: feel like this should be a function

File: src/site/artifacts/cargo_dist.rs [hunk 4]
```diff
@@ -183,10 +184,19 @@ pub fn build(config: &Config) -> Result<Box<div<String>>> {
...
+                    Ok(date) => date.format("%b %e %Y at %R UTC").to_string(),
+                    Err(_) => published_at.to_owned(),
+                };
+                let date_published = format!("Published at {}", formatted_date);
 
                 html.extend(html!(
                     <div class="hidden target artifact-header" data-targets=&targets>
-                        <h4>{text!("Install")}</h4>
+                        <h4>{text!(title)}</h4>
```
Comment (ashleygwilliams) [inline src/site/artifacts/cargo_dist.rs:197]: per my comment above i think we should add all this logic to the "install block" function

File: src/site/artifacts/page.rs [hunk 5]
```diff
@@ -7,7 +7,7 @@ use axohtml::html;
 
 pub fn build(config: &Config) -> Result<String> {
     let mut html = vec![];
-    let manifest = cargo_dist::fetch_manifest(config)?;
+    let (manifest, _) = cargo_dist::fetch_manifest(config)?;
```
Comment (ashleygwilliams) [inline src/site/artifacts/page.rs:10]: we should also put the data and version on the artifacts page :)

**Review by ashleygwilliams — CHANGES_REQUESTED** [review]

File: oranda-css/css/theme.css (hunk 1 above)
Comment (SaraVieira) [reply oranda-css/css/theme.css]: this file does not exist anymore

File: src/site/artifacts/cargo_dist.rs (hunk 3 above)
Comment (SaraVieira) [reply src/site/artifacts/cargo_dist.rs:189]: moved to a build_heading

File: src/site/artifacts/cargo_dist.rs (hunk 4 above)
Comment (SaraVieira) [reply src/site/artifacts/cargo_dist.rs:197]: done

File: src/site/artifacts/cargo_dist.rs (hunk 3 above)
Comment (SaraVieira) [reply src/site/artifacts/cargo_dist.rs:189]: broke layout so I stopped

Comment (ashleygwilliams) [reply src/site/artifacts/cargo_dist.rs:189]: hrm?

**Pull request merged**